### PR TITLE
global: test coverage and docs for documents

### PR DESCRIPTION
**Pull request by BadrAly**
* BETTER increase test coverage and add docs for documents.

Signed-off-by: Aly Badr <aly.badr@rero.ch>

### CODE REVIEW

File: rero_ils/modules/documents/views.py [hunk 1]
```diff
@@ -287,14 +287,15 @@ def abstracts_format(abstracts):
...
 
-@blueprint.app_template_filter()
-def item_library_locations(item_pid):
-    """Get the locations of the library of the given item."""
-    location_pid = Item.get_record_by_pid(item_pid)['location_pid']
-    location = Location.get_record_by_pid(location_pid)
-    library = Library.get_library_by_locationid(location.id)
-    locations = library.locations
-    return locations
+# @blueprint.app_template_filter()
```
Comment (jma) [inline rero_ils/modules/documents/views.py]: can we remove this?

File: rero_ils/modules/documents/views.py [hunk 2]
```diff
@@ -319,8 +320,8 @@ def item_status_text(item, format='medium', locale='en'):
     """Text for item status."""
     if item.available:
         text = _('available')
-        if item.dumps().get('item_type') == 'on_site_consultation':
-            text += ' ({0})'.format(_('on_site consultation'))
+        # if item.dumps().get('item_type') == 'on_site_consultation':
```
Comment (jma) [inline rero_ils/modules/documents/views.py]: can we remove this?

File: rero_ils/modules/documents/views.py (hunk 1 above)
Comment (BadrAly) [reply rero_ils/modules/documents/views.py]: done

File: rero_ils/modules/documents/views.py (hunk 2 above)
Comment (BadrAly) [reply rero_ils/modules/documents/views.py]: done

**Pull request merged**